ksql-engine/src/main/java/io/confluent/ksql/engine/InsertValuesExecutor.java

```diff
@@ -132,26 +135,57 @@ public void execute(
       final ConfiguredStatement<InsertValues> statement,
       final KsqlExecutionContext executionContext,
       final ServiceContext serviceContext
+  ) {
+    final InsertValues insertValues = statement.getStatement();
+    final KsqlConfig config = statement.getConfig()
+        .cloneWithPropertyOverwrite(statement.getOverrides());
+
+    final ProducerRecord<byte[], byte[]> record =
+        buildRecord(statement, executionContext, serviceContext, insertValues, config);
```

Comment:
nit: No need to pass `insertValues` or `config` as a parameter if you're already passing `statement`.

Reply:
This was to avoid doing the following in both buildRecord and execute, thus reducing duplication.

```
final InsertValues insertValues = statement.getStatement();

final KsqlConfig config = statement.getConfig()
        .cloneWithPropertyOverwrite(statement.getOverrides());
```

Reply:
Why?  the first is just a call to an accessor, the second slightly more.  I think it's better to have a cleaner method signature and two extra lines of code, than have a method signature that can be misused to save two lines of code.